plugins/vpp/ifplugin/data_resync.go

```diff
@@ -645,8 +644,7 @@ func (plugin *NatConfigurator) resolveMappings(nbDNatConfig *nat.Nat44DNat_DNatC
 				}
 				nbLocal := nbMapping.LocalIps[0]
 				vppLocal := vppMapping.LocalIps[0]
-				if nbLocal.LocalIp != vppLocal.LocalIp || nbLocal.LocalPort != vppLocal.LocalPort ||
-					nbLocal.Probability != vppLocal.Probability {
+				if *nbLocal == *vppLocal {
```

Comment:
shouldn't this be *nbLocal != *vppLocal ?

Reply:
yes it should, thank you